src/main/java/seedu/address/logic/commands/NoteCommand.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NOTE;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Maintainer;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Note;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Staff;
+import seedu.address.model.person.Supplier;
+
+/**
+ * Adds a note of an existing person in the address book.
+ * A non-empty note must be specified.
+ */
+public class NoteCommand extends Command {
+
+    public static final String MESSAGE_ADD_NOTE_SUCCESS = "Added note to Person: %1$s";
+    public static final String COMMAND_WORD = "/note";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds note to person.\n"
+            + "Parameters: "
+            + PREFIX_NAME + "Moochie"
+            + PREFIX_NOTE + "Meet at 6pm tuesday"
+            + "Example: " + COMMAND_WORD + " Moochie" + "Meet at 6pm tuesday";
```

Comment:
Wait I'm so confused, shouldn't this line be: 
"Example: " + COMMAND_WORD + PREFIX_NAME + " Moochie" + PREFIX_NOTE + "Meet at 6pm Tuesday";